waku/waku_core/subscription/subscription_manager.nim

```diff
@@ -0,0 +1,48 @@
+when (NimMajor, NimMinor) < (1, 4):
+  {.push raises: [Defect].}
+else:
+  {.push raises: [].}
+
+import
+  std/tables,
+  stew/results,
+  chronicles,
+  chronos
+
+import
+  ./push_handler,
+  ../topics,
+  ../message
```

Comment:
Maybe any of these imports can be deleted from `waku/waku_filter/client.nim`

Reply:
I can check if there is any unused ones.

Reply:
There is no unused import in this client.nim - surprisingly